### DISCUSSION TITLE
v1.22.2 release

### DISCUSSION THREAD

**Discussion by andyzhangx** [kubernetes-sigs/blob-csi-driver#1012] [Announcements]
## What's Changed
* [release-1.22] fix: CVE-2023-2431 by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/969
* [release-1.22] fix: storage account search default values by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/970
* [release-1.22] cleanup: remove secret print in error message by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/974
* fix: CVE-2023-2727 by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/982
* [release-1.22] feat: install blobfuse 2.0.4 as default version by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/981
* [release-1.22] feat: install blobfuse 2.0.5 as default version by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/989
* [release-1.22] cleanup: add secret name in the log by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/992
* [release 1.22] feat: support aznfs mount helper by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1006
* [release-1.22] chore: set enable-aznfs-mount default value as false by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1011
* doc: cut v1.22.2 release by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1007

**Full Changelog**: https://github.com/kubernetes-sigs/blob-csi-driver/compare/v1.22.1...v1.22.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.22.2'>v1.22.2 release</a>.</em>

**Comment by calohmn** [comment]
@andyzhangx It seems to me the release isn't complete yet as the v1.22 helm chart files are only in branch `release-1.22` yet, not in `master`.
The "install by [helm charts](https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/charts)" link in the README leads to the `master` branch files, so I thought one is supposed to reference the chart from `master`.

**Reply by andyzhangx** [reply]
https://github.com/kubernetes-sigs/blob-csi-driver/pull/1014 fixed this issue, can you try again? thanks.

**Reply by calohmn** [reply]
Thanks, it works now.
